README.md

```diff
@@ -11,6 +11,31 @@ issues, thanks to the language's guarantees.
 
 ## Usage
 
+The [user guide][UG] provides an overview of everything available in the 
+project. It can be generated offline using **mdbook**: 
+
+```shell
+# Serve the doc on a local server
+mdbook serve --open -d ../target/doc/ honeycomb-guide/ &
+cargo doc --all --no-deps
+
+# Kill the local server
+kill $(pidof mdbook) 
+
+# Without pidof
+kill $(ps -e | awk '/mdbook/ {print $1}')
+```
+
+You may also use the `build` subcommand of mdbook and browse the files directly,
+but there may be navigation problems because of some implicit links (e.g. folders
+instead of index files) depending on your browser.
```

Comment:
Not sure either, but you can leave it there.

Reply:
I removed it since there already is a linking warning in the notes from the honeycomb-guide section of the UG